client/gutenberg/extensions/publicize/form.jsx

```diff
@@ -23,12 +23,14 @@ const PublicizeForm = compose( [
 	withSelect( select => {
 		const meta = select( 'core/editor' ).getEditedPostAttribute( 'meta' );
 		const postTitle = select( 'core/editor' ).getEditedPostAttribute( 'title' );
-		const message = get( meta, [ 'jetpack_publicize_message' ], '' ) || postTitle || '';
+		const message = get( meta, [ 'jetpack_publicize_message' ], '' );
 
 		return {
 			connections: select( 'core/editor' ).getEditedPostAttribute(
 				'jetpack_publicize_connections'
 			),
+			meta,
```

Comment:
Oops, seems like this sneaked its way back in during the rebase. No longer needed per #28689.

Reply:
👍 Removed again in https://github.com/Automattic/wp-calypso/pull/28695/commits/71897bda68ebef092129f321e63d17f630ac08f1